kagenti/ui/lib/agent_details_page.py

```diff
@@ -221,6 +220,9 @@ def render_agent_details_content(agent_k8s_name: str):
 
     agent_chat_name_for_sdk = agent_k8s_name
 
+    # Show access token
+    st.session_state[constants.TOKEN_STRING][constants.ACCESS_TOKEN_STRING]
```

Comment:
Here, I'm trying to show the access token before we call `_handle_chat_interaction()`